packages/forklift-console-plugin/src/modules/Plans/views/details/components/PlanPageHeadings.tsx

```diff
@@ -19,7 +23,8 @@ import {
   V1beta1StorageMap,
 } from '@kubev2v/types';
 import { useK8sWatchResource } from '@openshift-console/dynamic-plugin-sdk';
-import { Level, List, ListItem, PageSection } from '@patternfly/react-core';
+import { Button, Level, List, ListItem, PageSection, Tooltip } from '@patternfly/react-core';
+import CalendarIcon from '@patternfly/react-icons/dist/esm/icons/calendar-alt-icon';
```

Comment:
please change this import to be import {CalendarIcon} from '@patternfly/react-icons' and not the dist directory